<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HealthCareWorkerLoginPage</class>
 <widget class="QWidget" name="HealthCareWorkerLoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Electronic Vaccination Data System</string>
  </property>
  <widget class="QLabel" name="lb_username">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="et_hcwUsername">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>User Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="et_hcwPass">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>*****</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_hcwSbmitCred">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>S u b m i t</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_hcwlginun">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Incorrect username Entered, please try again.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_hcwlginpw">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Incorrect password Entered, please try again.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_hcwlgmissingnfo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>All the Fields are compulsory, please fill in all of them.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_hcwlgnErr">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>259</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Something went wrong please contact the system administrator</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
